package.json

```diff
@@ -48,7 +48,10 @@
         "eslint-plugin-prettier": "^5.0.1",
         "eslint-plugin-simple-import-sort": "^10.0.0",
         "eslint-plugin-unused-imports": "^3.0.0",
+        "fast-glob": "3.3",
```

Comment:
what's the point of adding this dependency? your use case is just `glob("./dir/*")` which could be replaced with a simple `fs.readdir`

Reply:
Fair point, I'll refactor things